### ISSUE TITLE
Flutter framework and cocoapods

### ISSUE THREAD

**Issue by vitor-gyant** [CocoaPods/CocoaPods#8917]
Hello All,

I'm trying to encapsulate Flutter inside a framework. The problem I'm facing now is that the pod install command does not generate the -framework.sh and xcfilelist files that I need to make sure all the pod dependencies frameworks are copied into my framework otherwise, the app launch fails with dyld: Library not loaded dynamic framework.

Could someone kindly help me figure out how to achieve this ? 

Basically, I'm missing the Embed Frameworks phase in my framework target but don't know why ...

Thanks in advance,

**Comment by dnkoutso** [comment]
This is not a report we can work with. We need a reproducible example and also this might be a flutter integration issue.

I am going to close this but I will re-open once clear concise steps of the issue are posted hopefully with a sample app.
